colab/demo.ipynb

```diff
@@ -58,59 +61,57 @@
         "* **WARNING: Running pip as the 'root' user...:**:\n",
         "    * This and other pip warnings or errors can be safely ignored.\n",
         "* **Other problems?**\n",
-        "    * Feel free to create an issue on our [GitHub repo](https://github.com/nerfstudio-project/nerfstudio).\n"
+        "    * Feel free to create an issue on our [GitHub repo](https://github.com/nerfstudio-project/nerfstudio)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "## Nerfstudio Demo"
       ]
     },
     {
       "cell_type": "code",
       "execution_count": null,
       "metadata": {
-        "id": "9oyLHl8QfYwP",
-        "cellView": "form"
+        "cellView": "form",
+        "id": "9oyLHl8QfYwP"
       },
       "outputs": [],
       "source": [
-        "#@markdown <h1>Install Nerfstudio and Dependencies (~8 min)</h1>\n",
+        "#@markdown <h3>Install Nerfstudio and Dependencies (~8 min)</h3>\n",
         "\n",
         "%cd /content/\n",
         "!pip install --upgrade pip\n",
         "!pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 --extra-index-url https://download.pytorch.org/whl/cu118\n",
```

Comment:
```suggestion
        "!pip install torch==2.0.1+cu118 torchvision==0.15.2+cu118 torchaudio==2.0.2+cu118  --extra-index-url https://download.pytorch.org/whl/cu118\n",
```

I also needed to pin the version of torchaudio to get training to start on Google Colab. I only tested it on Google Colab and as of today.